Components/Optimizers/AdaptiveStochasticVarianceReducedGradient/itkStochasticVarianceReducedGradientDescentOptimizer.h

```diff
@@ -191,8 +190,8 @@ class StochasticVarianceReducedGradientDescentOptimizer : public ScaledSingleVal
   StopConditionType m_StopCondition{ MaximumNumberOfIterations };
   DerivativeType    m_PreviousGradient{};
   // DerivativeType                m_PrePreviousGradient;
-  ParametersType        m_PreviousPosition{};
-  ThreaderType::Pointer m_Threader{ ThreaderType::New() };
+  ParametersType             m_PreviousPosition{};
+  MultiThreaderBase::Pointer m_Threader{ MultiThreaderBase::New() };
```

Comment:
Might use thread pools, so might _again_ improve the performance of StochasticVarianceReducedGradientDescentOptimizer 👍